src/main/resources/static/gogo_components/pokemon-view/pokemon-view.html

```diff
@@ -14,7 +14,8 @@
             <div class="thumbnail">
                 <img src="https://img.pokemondb.net/artwork/{{pokemon.pokedexLink}}.jpg" style="height: 200px;"/>
                 <div class="caption">
-                    <h3>{{pokemon.name}}</h3>
+                    <h3>{{pokemon.nickname}}</h3>
```

Comment:
That was actually the first idea, but then we would have some boxes with smaller sizes (or bigger, if you are a normal person and don't rename every pokemon). Unless it is done in the same line.